experience/resume.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title> Resume & Honors!</title>
+		<link rel="stylesheet" type="text/css" href="../css/style.css">
```

Comment:
:hammer: The css file isn't loading. The `../` tells the computer to _look up_ to the parent directory _**relative**_ to where this index.html file lives. There isn't actually a parent folder there. You want to just look in the css folder then grab styles.css.